File: Python/GCD_usingRecurssion.py
# Recursive function to find GCD
def gcd(a, b):
    if b == 0:
        return a
    return gcd(b, a % b)

# Main function
def main():
    num1, num2 = map(int, input("Enter two numbers: ").split())
    print(f"GCD of {num1} and {num2} is: {gcd(num1, num2)}")

# Run the program
if __name__ == "__main__":
    main()

File: C/GCD_using_recurssion.c
#include <stdio.h>

// Recursive function to find GCD
int gcd(int a, int b) {
    if (b == 0) {
        return a;
    }
    return gcd(b, a % b);
}

int main() {
    int num1, num2;
    
                                                                // Input two numbers
    printf("Enter two numbers: ");
    scanf("%d %d", &num1, &num2);
    
                                                                // Call the gcd function and display the result
    printf("GCD of %d and %d is: %d\n", num1, num2, gcd(num1, num2));
    
    return 0;
}
